source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -242,40 +278,76 @@ void CGLTFMeshFileLoader::MeshExtractor::copyNormals(
 	
 	for (std::size_t i = 0; i < count; i++) {
 		const auto n = readVec3DF(BufferOffset(buffer,
-			3 * sizeof(float) * i));
+			3 * sizeof(float) * i), core::vector3df{1,1,1});
 		vertices[i].Normal = n;
 	}
 }
 
+/**
+ * Streams texture coordinate raw data into usable buffer via reference.
+ * Buffer: ref Vector<video::S3DVertex>
+*/
 void CGLTFMeshFileLoader::MeshExtractor::copyTCoords(
 		const std::size_t accessorIdx,
 		std::vector<vertex_t>& vertices) const
 {
+
 	const auto& buffer = getBuffer(accessorIdx);
 	const auto count = getElemCount(accessorIdx);
 
+	// Reference material for Josiah.
+	// const auto normalized = isAccessorNormalized(accessorIdx);
+	// const auto acc = m_model.accessors[accessorIdx];
+	// const auto min = acc.minValues;
+	// const auto max = acc.maxValues;
+

```

Comment:
Why are you committing unused code and then expecting me to finish the implementation? Please send me the commit and let me work with that. I'm not going to merge stuff like this into the remote branch.